src/main/java/org/venice/beachfront/bfapi/services/IABrokerService.java

```diff
@@ -83,8 +89,34 @@ public Scene getScene(String sceneId, String planetApiKey, boolean withTides) th
 				throw new IABrokerUnknownException(response.toString());
 			}
 		}
+		
+		JsonNode responseJson = response.getBody();
+
+		Scene scene = new Scene();
+		scene.setRawJson(responseJson);
+		scene.setSceneId(responseJson.get("id").asText());
+		scene.setCloudCover(responseJson.get("properties").get("cloudCover").asDouble());
+		scene.setResolution(responseJson.get("properties").get("resolution").asInt());
+		scene.setCaptureTime(DateTime.parse(responseJson.get("properties").get("acquiredDate").asText()));
+		scene.setSensorName(responseJson.get("properties").get("sensorName").asText());
+		scene.setUri(responseJson.get("properties").get("location").asText());
```

Comment:
These parses should be contained in a single try-catch at the very least.  Parsing errors are very likely when you go three-properties deep in a single line. It's fine - just put up a try-catch so we can have a nicely formatted exception as to what a parsing error might be.